oasislmf/preparation/gul_inputs.py

```diff
@@ -376,24 +397,26 @@ def get_gul_input_items(
             gul_inputs_df.rename(columns=hazard_group_id_cols_map)[sorted(list(hazard_group_id_cols_map.values()))], index=False).to_numpy() >> 33
     ).astype('uint32')
 
-    # Select only required columns
-    # Order here matches test output expectations
+    # select only the required columns
+    # the output order matches test output expectations
     usecols = (
         ['loc_id', portfolio_num, acc_num, loc_num] +
         ([SOURCE_IDX['loc']] if SOURCE_IDX['loc'] in gul_inputs_df else []) +
         ['peril_id', 'coverage_type_id', 'tiv', 'areaperil_id', 'vulnerability_id'] +
         terms +
         (['model_data'] if 'model_data' in gul_inputs_df else []) +
-        ['is_bi_coverage', 'group_id', 'coverage_id', 'item_id', 'status']
+        ['is_bi_coverage', 'group_id', 'coverage_id', 'item_id', 'status', 'disagg_id', 'NumberOfBuildings', 'IsAggregate',] +  # disagg_id is needed for fm_summary_map
+        (["peril_correlation_group", "damage_correlation_value", 'hazard_group_id', "hazard_correlation_value"] if correlations is True else [])
     )
-    if correlations is True:
-        usecols += ["peril_correlation_group", "damage_correlation_value", 'hazard_group_id', "hazard_correlation_value"]
 
-    usecols = [col for col in usecols if col in gul_inputs_df]
-    gul_inputs_df = gul_inputs_df[usecols]
+    usecols = list(set(usecols) & set(gul_inputs_df.columns))
```

Comment:
I think this give a different order for the columns and make the header of gul_summary different